src/test/java/TodoTest.java

```diff
@@ -0,0 +1,27 @@
+package test.java;
+
+import main.java.Todo;
+import main.java.Task;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+public class TodoTest {
```

Comment:
Similar to what @coderhuang559 has mentioned for your `Todo` class, perhaps your `TodoTest` class would also have to be in Pascal Case.